cmd/geth/verkle.go

```diff
@@ -75,7 +75,7 @@ func checkChildren(root verkle.VerkleNode, resolver verkle.NodeResolverFn) error
 	switch node := root.(type) {
 	case *verkle.InternalNode:
 		for i, child := range node.Children() {
-			childC := child.ComputeCommitment().Bytes()
+			childC := child.Commitment().Bytes()
```

Comment:
Mentioning something that might be off-topic for this PR, but I wonder if it might be interesting at some point to unify `Commitment()` and `Commit()`. 

Mostly because [`Commitment()` panics if there's no value in the cache](https://github.com/gballet/go-verkle/blob/master/tree.go#L804), and as a client, it might not be entirely obvious when that invariant holds.

So, something like a unique `Commit()` method that checks the cache, and whenever some action touches that node, it sets that to `nil` again, so on the next `Commit()` that is re-computed.

I feel you intentionally created these `Commit()` and `Commitment()` APIs, so there is probably a good reason I can be missing. Maybe it's useful to access the previous commitment value even after touching the node.

I'm just surfacing some personal uncertainty I had while deciding between `Commit()` and `Commitment()` since it wasn't entirely obvious and could produce a panic.